frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -64,12 +64,35 @@ export interface OngoingMissionType {
     [missionId: string]: boolean
 }
 
-export const compareInspections = (i1: Inspection, i2: Inspection) => {
-    if (!i1.missionDefinition.inspectionFrequency) return 1
-    if (!i2.missionDefinition.inspectionFrequency) return -1
-    if (!i1.missionDefinition.lastRun) return 1
-    if (!i2.missionDefinition.lastRun) return -1
-    else return i1.deadline!.getTime() - i2.deadline!.getTime()
+/**
+ * Compares two inspections so that they can be sorted. The inspection which
+ * has an inspection frequency and/or last run while the other doesn't will
+ * appear first. If these values are set for both inspections then the
+ * inspection with the closest deadline will appear first.
```

Comment:
```suggestion 
 * Compares two inspections so that they can be sorted. The inspection with
 * an inspection frequency and/or a last run will be sorted before an inspection without.
 * If both inspections have an inspection frequency and/or a last run, the
 * inspection with the closest deadline will be sorted above the other.